tests/Fixtures/CDN/rewrite.html

```diff
@@ -515,19 +515,31 @@ <h1 class="entry-title">Page Image Alignment</h1>		</header><!-- .entry-header -
 <figure class="post-thumbnail">
 	<img width="1568" height="1046" src="http://cdn.example.org/wp-content/uploads/2018/03/sticker-mule-189122-unsplash-1568x1046.jpg" class="attachment-post-thumbnail size-post-thumbnail wp-post-image" alt="" srcset="http://example.org/wp-content/uploads/2018/03/sticker-mule-189122-unsplash-1568x1046.jpg 1568w, http://example.org/wp-content/uploads/2018/03/sticker-mule-189122-unsplash-300x200.jpg 300w, http://example.org/wp-content/uploads/2018/03/sticker-mule-189122-unsplash-768x513.jpg 768w, http://example.org/wp-content/uploads/2018/03/sticker-mule-189122-unsplash-1024x683.jpg 1024w" sizes="(max-width: 1568px) 100vw, 1568px" />
 </figure>
+<picture>
+	<source srcset="http://cdn.example.org/images/site/logo/logo-acommeassure.webp" type="image/webp">
+	<source srcset="/images/site/logo/logo-acommeassure.png 100w" type="image/png">
```

Comment:
Shouldn't this line be:
`<source srcset="http://cdn.example.org/images/site/logo/logo-acommeassure.png 100w" type="image/png">`

Reply:
No, the method will not match this one because of the ` 100w` in the `srcset`. But it matches the first one because there is no width definer.

This `srcset` will be taken care of by `rewrite_srcset()` in the end.

Reply:
Ok. Got it. Thanks for the clarification